### PR TITLE
Latest cloudposse features

### PR DESCRIPTION
**Pull request by okgolove**
## what
* Allow Terraform 0.14
* Use gte for providers
* Use latest label module and context feature

## why
* The module should have context feature due to it is used by another modules like ecs-web-app

### CODE REVIEW

**Comment by joe-niland** [comment]
Hi @okgolove while you're at it, could you please integrate these changes also?
https://github.com/cloudposse/terraform-aws-ecs-cloudwatch-sns-alarms/pull/18

File: .github/workflows/chatops.yml [hunk 1]
```diff
@@ -9,13 +9,13 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: "Handle common commands"
-        uses: cloudposse/actions/github/slash-command-dispatch@0.15.0
+        uses: cloudposse/actions/github/slash-command-dispatch@0.16.0
```
Comment (joe-niland) [inline .github/workflows/chatops.yml:12]: We can manually correct this version to 0.21.0

**Comment by joe-niland** [comment]
Thanks, and for the tests:

```
cd test/src
rm Gopkg.*
go mod init github.com/cloudposse/$(cd ../../ && basename `pwd`)
go mod tidy
```

**Comment by joe-niland** [comment]
@okgolove just FYI, you can run `make readme` locally to avoid the rebuilding here

**Comment by joe-niland** [comment]
@okgolove seems to be a linting error. Please try running terraform fmt for / and /examples...

**Review by joe-niland — APPROVED** [review]

**Comment by joe-niland** [comment]
Thanks @okgolove we got there in the end!

**Comment by okgolove** [comment]
@joe-niland hurrah! Thank you :)

**Pull request merged**